.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,15 @@
+**Is your feature request related to a problem?**
```

Comment:
The issue in `documentation-website` will be created with this content. Maybe use this template https://github.com/opensearch-project/OpenSearch/pull/2929/files#diff-cafe2730d029a63a9cc0fa0d9f853fca6811942e8fea2ba3b13ec08ae038dfdb so the static content is relevant?

Reply:
Sure, I just used the current issue template for the documentation-website, but I'll make it more relevant

Reply:
Sure, I just used the current issue template for the documentation-website, but I'll make it more relevant